jsapp/js/components/processing/analysis/constants.ts

```diff
@@ -16,6 +16,10 @@ export type AnalysisQuestionType =
   | 'qual_tags'
   | 'qual_text';
 
+// We use it on other parts of code that is not related to analysis and is not
```

Comment:
I'm not sure what this comment means, I feel like it should say something like "We don't need to show qualitative note questions outside of analysis"? Or maybe I'm misunderstanding

Reply:
I wanted to justify having a `const` here, while that `AnalysisQuestionType` should be more than enough. I rewrote the comment, but maybe the best way is to not have a comment at all here?

My thinking right now is that before TypeScript we had these long elaborate enums to ensure every piece of code is using the same strings. After moving some code to TS it become clear, that having a type is safe enough, and keeping const/enums for list of names of things is overdoing things.